code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -361,10 +360,8 @@ var/const/INGEST = 2
 		del_reagent(R.id)
 	return 0
 
-/datum/reagents/proc/reaction_check(mob/M, datum/reagent/R)
+/datum/reagents/proc/reaction_check(mob/living/M, datum/reagent/R)
 	var/can_process = 0
-	if(!istype(M, /mob/living))		//Non-living mobs can't metabolize reagents, so don't bother trying (runtime safety check)

```

Comment:
`reaction_check` is only ever called in `reaction`, which already has `isliving` checks, so this is redundant.